modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/tests/PublishedNodes/CanSendAlarmToIoTHubTest.json

```diff
@@ -0,0 +1,19 @@
+﻿[
+  {
+    "EndpointUrl": "opc.tcp://localhost:52210/UA/SampleServer",
```

Comment:
To confirm, you use a fixed port, so this always works?

Reply:
Yep, it works as all tests use the same server/port in a class. If we split the tests into multiple classes, it should still be fine as tests are not run in parallel unless tests are configured for parallism (IMHO). If we do run into a situation where we need to have unique ports, we can inject port as param to server fixture and base class.